_data/db/administrations/nt.yml

```diff
@@ -0,0 +1,6 @@
+---
+code: nt
+name:
+  en: Territory of Northwest Territories
+  fr: Territoire des Territoires du Nord-Ouest
```

Comment:
it should be Northwest Territories (Territoires du Nord-Ouest)

Reply:
I di however noticed that all provinces had the 'Province of' prefix, and all territories had the 'Territory of' prefix. So maybe changing it would remove some consistency, or maybe we should remove all of the prefixes?